### ISSUE TITLE
Enhancement | Write page

### ISSUE THREAD

**Issue by aadeshkulkarni** [aadeshkulkarni/medium-app#46]
### Current:

<img width="1793" alt="Screenshot 2024-05-14 at 9 42 20 PM" src="https://github.com/aadeshkulkarni/medium-app/assets/67855887/7442adeb-9d45-4f60-aa13-32cd494602f6">

### Expectation:

Similiar to https://medium.com/p/3434770f0676/edit

- Title should have a placeholder 'Tell your story', formatting should be like shown below (Ignore the + icon for now)
- The text editor should not have a border, should be centred.
- Publish button should be at the top instead of bottom.
- Menu and Notification icons are both out of scope of this ticket.
- Autosave / Draft feature is out of scope (but needs to be implemented in future)

<img width="1793" alt="Screenshot 2024-05-14 at 9 39 31 PM" src="https://github.com/aadeshkulkarni/medium-app/assets/67855887/230f2689-78cb-48d9-8144-6c444795a1ef">

**Comment by KrupaPanchal2527** [comment]
I would love to work on this. Please assign to me if possible.

**Comment by aadeshkulkarni** [comment]
@KrupaPanchal2527 - This has been assigned to you. Would be grateful if could estimate the timeline for this one and share it.

**Comment by aadeshkulkarni** [comment]
Also, #24 is related to this one. It would be awesome if you could hit 2 targets with 1 arrow.

**Comment by KrupaPanchal2527** [comment]
@aadeshkulkarni I can submit the changes for this one by or before next Friday. Would that be okay? I can rearrange some priorities if it needs to be finished before that. Just let me know.

**Comment by KrupaPanchal2527** [comment]
> Also, #24 is related to this one. It would be awesome if you could hit 2 targets with 1 arrow.

Once the UI revamp for the Write page is done, I'll do a bit of research on other libraries and will share a list and findings with you so that we can make a decision on which library to bring in next.

**Comment by aadeshkulkarni** [comment]
> @aadeshkulkarni I can submit the changes for this one by or before next Friday. Would that be okay? I can rearrange some priorities if it needs to be finished before that. Just let me know.

There is no hurry. This works.

**Comment by aadeshkulkarni** [comment]
> > Also, #24 is related to this one. It would be awesome if you could hit 2 targets with 1 arrow.
> 
> Once the UI revamp for the Write page is done, I'll do a bit of research on other libraries and will share a list and findings with you so that we can make a decision on which library to bring in next.

sounds good. Thanks @KrupaPanchal2527 :)

**Comment by KrupaPanchal2527** [comment]
@aadeshkulkarni Have opened [a PR](https://github.com/aadeshkulkarni/medium-app/pull/65) for this enhancement. 

I don't think we might need to change the text editor library based on the results I have seen with the current one. You can review the PR or take a look at the screenshots attached to see how the current quill editor looks like. Then we can discuss whether we need to bring in a new text editor library or not.

**Comment by aadeshkulkarni** [comment]
I agree, no need for #24 anymore. Keep up the good work 🚀
